gitpod-io/gitpod
# Remove gitpod-messagebus component WEB-618

## Description


## Related Issue(s)

* Depends on https://github.com/gitpod-io/gitpod/pull/18216

## How to test


CI passes. Messagebus is no longer deployed (as of https://github.com/gitpod-io/gitpod/pull/18216) so there's no need for a preview build

## Documentation


#### Preview status

gitpod:summary

## Build Options

<details>
<summary>Build</summary>

      Run the build with werft instead of GHA
      Run Leeway with `--dont-test`
</details>

<details>
<summary>Publish</summary>

</details>

<details>
<summary>Installer</summary>

  Add desired feature flags to the end of the line above, space separated
</details>

<details>
<summary>Preview Environment</summary>

      If enabled this will build `install/preview`
      If enabled this will create the environment on GCE infra
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`
</details>

/hold
